README.md

```diff
@@ -180,23 +180,23 @@ depending on whether you want to use HTTPS or SSH. Finally, run the
 following:
 
 ```sh
```

Comment:
How about changing `sh` to `bash`?

Reply:
will this effectively change anything at all?